tests/UI/package.json

```diff
@@ -111,7 +111,7 @@
   "homepage": "https://github.com/PrestaShop/PrestaShop/tree/develop/tests/UI#readme",
   "dependencies": {
     "@faker-js/faker": "^9.0.3",
-    "@prestashop-core/ui-testing": "https://github.com/PrestaShop/ui-testing-library#main",
+    "@prestashop-core/ui-testing": "https://github.com/jolelievre/ui-testing-library#ui-tests-classic",
```

Comment:
Please remove this change, please

Reply:
It's temporary, I need to valide the changes from https://github.com/PrestaShop/ui-testing-library/pull/535 first if it works and you agree with the changes we can merge it and I'll update this PR